model/annotations/task_annotations.go

```diff
@@ -25,6 +25,16 @@ type TaskAnnotation struct {
 	SuspectedIssues []IssueLink `bson:"suspected_issues,omitempty" json:"suspected_issues,omitempty"`
 	// links to tickets created from the task using a custom web hook
 	CreatedIssues []IssueLink `bson:"created_issues,omitempty" json:"created_issues,omitempty"`
+	// non-ticket links to be displayed in the UI metadata sidebar
+	TaskLinks []TaskLink `bson:"task_links,omitempty" json:"task_links,omitempty"`
```

Comment:
nit: technically these could still be tickets, so I would just say links in the comment. Also I know I used task_links in the ticket but maybe MetadataLinks is more specific? Whatever works though

Reply:
Sure, fine to rename to MetadataLinks